src/tracing/trace.ts

```diff
@@ -24,9 +24,11 @@ function stringToHeader(value: string): StringDict {
 export function createTracerProvider(
   otlpEndpoint: string,
   otlpHeaders: string,
+  otelServiceName: string,
   workflowRunJobs: WorkflowRunJobs
```

Comment:
```suggestion
export function createTracerProvider(
  otlpEndpoint: string,
  otlpHeaders: string,
  workflowRunJobs: WorkflowRunJobs,
  otelServiceName?: string | null | undefined,
```
Lets make this a non-breaking change to the API.